core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2047,10 +2047,6 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
    @Message(value = "ClusterLoader has been deprecated and will be removed in a future version with no direct replacement", id = 595)
    void warnUsingDeprecatedClusterLoader();
 
-   @LogMessage(level = WARN)
```

Comment:
Instead of removing this code, use a block-comment. This reminds us why there is a gap in the logging IDs.